src/provider/mod.rs

```diff
@@ -890,15 +890,10 @@ async fn send_blob<W: AsyncWrite + Unpin + Send + 'static>(
             // 'static lifetime.
             writer = tokio::task::spawn_blocking(move || {
                 let file_reader = std::fs::File::open(&path)?;
-                let outboard_reader = std::io::Cursor::new(outboard);
                 let mut wrapper = SyncIoBridge::new(&mut writer);
```

Comment:
Can we not get rid of the `SyncIoBridge` now if we have async support?

Reply:
It si gone now.